src/encryptdecrypt/cryptors/AbstractCryptor.java

```diff
@@ -0,0 +1,22 @@
+package encryptdecrypt.cryptors;
+
+import java.io.IOException;
+
+public abstract class AbstractCryptor {
+
+    public void start() {
+        try {
+            prepareData();
+            processData();
+            outputData();
+        } catch (IOException e) {
+            System.out.println("Error");
+        }
+    }
+
+    protected abstract void prepareData() throws IOException;
+
+    protected abstract void processData();
+
+    protected abstract void outputData() throws IOException;
```

Comment:
What is output data? again question, does "cryptor" use only once?

Reply:
- [X] Fixed